website/docs/api/internal/internal-backstage-api.md

```diff
@@ -7,7 +7,9 @@ title: /internal-backstage/prometheus
 
 `GET http://unleash.host.com/internal-backstage/prometheus`
 
-Unleash uses prometheus internally to collect metrics. These are available on the given url if the `serverMetrics` option is enabled (default=true).
+Unleash uses Prometheus internally to collect metrics. These are available on the given url if the `serverMetrics` option is enabled (default=true).
+
+Note that it's not recomended to expose Prometheus metrics to the public. You should therefore drop all external access to `/internal-backstage/*` on the network layer to keep your instance secure. 
```

Comment:
I think adding this bit is a very good suggestion! 💯  However, I'd like to ask you to add a little bit more nuance to it to make it easier to read.

A couple things:
1. do you have a reference for why it's not recommended to expose prometheus metrics? Or can you explain why it's not recommended? It might be obvious to you, but it's not to everyone else.
2. "You should therefore drop all external access to `/internal-backstage/*` on the network layer to keep your instance secure." Could this be something like: 
3. 
```suggestion
Note that it's not recommended to expose Prometheus metrics to the public. Thus, if you want to keep metrics enabled, you should block all external access to `/internal-backstage/*` on the network layer to keep your instance secure. 
```

What do you think?